client/src/livingdocs-component-app/app.css

```diff
@@ -171,34 +171,38 @@ html {
   text-decoration: none;
 }
 
-.livingdocs-component-item-preview,
-.livingdocs-component-item-options,
-.livingdocs-component-item-controls {
+.livingdocs-component-item-preview-container,
+.livingdocs-component-item-controls-container {
   width: 90%;
   margin: var(--q-space-base) auto 0 auto;
 }
 
 .livingdocs-component-item-options-container {
+  overflow-y: scroll;
```

Comment:
currently the scrollbar is always visible. From what I see from a short test in browser if this is `overflow: auto` this should work as expected: the scrollbar is visible when needed.
Is there a specific reason why the scrollbar should always be there?

Reply:
Ah I understand. Yes thats the behavior I want. Fixed in https://github.com/nzzdev/Q-editor/pull/169/commits/56b9f1f8efbc927e323c443684ac1ad9b4f30868